scripts/generator/graphql_generator/csharp/Cart.CheckoutAccessors.cs.erb

```diff
@@ -5,6 +5,19 @@ namespace <%= namespace %> {
     using <%= namespace %>.GraphQL;
 
     public partial class Cart {
+
+        /// <summary>
+        /// Retrieve the currency code for the current checkout.
+        /// </summary>
+        /// <exception cref="NoCheckoutException">Calling this accessor before a checkout was created for the Cart. See Cart.GetWebCheckoutLink</exception>
+        public CurrencyCode GetCurrencyCode() {
```

Comment:
I've also wondered if we should just expose `CurrentCheckout` which would be another way to do this.